### PR TITLE
fix: history summary

### PR DESCRIPTION
**Pull request by raffaellasuardini**
## Description

As mentioned in the issue, the left padding in the page history has been updated.

```
Before:
#change-history .paginator,.change-list table .paginator,.dashboard table .paginator,.results table .paginator,.tabular table .paginator {
    box-shadow: none!important;
}
```

After:
```
#change-history .paginator,.change-list table .paginator,.dashboard table .paginator,.results table .paginator,.tabular table .paginator {
    box-shadow: none!important;
    padding-left: 10px;
    box-sizing: border-box;
}
```

* #516

### CODE REVIEW

**Comment by raffaellasuardini** [comment]
Sure @fsbraun. I'm going to work on it.

File: djangocms_admin_style/sass/components/_tables.scss [hunk 1]
```diff
@@ -292,6 +292,8 @@
     }
     .paginator {
         box-shadow: none !important;
+        padding-left: 10px;
```
Comment (fsbraun) [inline djangocms_admin_style/sass/components/_tables.scss:295]:
```suggestion
        padding-inline-start: 10px;
```
With pseudo properties we can automatically support RTL languages.

**Comment by fsbraun** [comment]
@raffaellasuardini Can you **also** run `gulp sass` to update the css assets?

**Comment by raffaellasuardini** [comment]
@fsbraun I ran the gulp file but it also created files like` .eot .ttf `and `.woff` should I also include them?

**Comment by fsbraun** [comment]
Yes, please do that. Sometimes updates on the compiler lead to minor changes in the binaries.

**Review by fsbraun — APPROVED** [review]

**Pull request merged**